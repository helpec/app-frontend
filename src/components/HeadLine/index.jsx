import React, { Component } from 'react';

class HeadLine extends Component {
  render() {
    return (
        <section id="inner-headline">
          <div className="container">
            <div className="row">
              <div className="span4">
                <div className="inner-heading">
                  <h2>{this.props.title}</h2>
                </div>
              </div>
            </div>
          </div>
        </section>
    )
  }
}

export default HeadLine;