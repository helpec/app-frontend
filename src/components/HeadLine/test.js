import React from 'react';
import renderer from 'react-test-renderer';

import HeadLine from '.';

test('Testar componente de HeadLine da aplicação', () => {

    const component = renderer.create(
        <HeadLine title="Teste componente"/>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});
