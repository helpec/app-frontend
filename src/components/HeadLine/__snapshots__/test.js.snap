// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testar componente de HeadLine da aplicação 1`] = `
<section
  id="inner-headline"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="span4"
      >
        <div
          className="inner-heading"
        >
          <h2>
            Teste componente
          </h2>
        </div>
      </div>
    </div>
  </div>
</section>
`;
